gobrew.go

```diff
@@ -671,42 +676,64 @@ func (gb *GoBrew) getGithubTags(repo string) (result []string) {
 
 	githubTags = make(map[string][]string, 0)
 	client := &http.Client{}
-	url := "https://api.github.com/repos/kevincobain2000/gobrew/git/refs/tags"
-	if repo == "golang/go" {
+	url := goTagsApi
+	if repo == golangRepo {
 		url = goBrewTagsApi
 	}
+
 	request, err := http.NewRequest("GET", url, nil)
 	if err != nil {
 		utils.Errorf("[Error] Cannot create request: %s", err)
 		return
 	}
 
-	request.Header.Set("User-Agent", "gobrew")
+	request.Header.Set("User-Agent", gobrewRepo)
 
 	response, err := client.Do(request)
 	if err != nil {
 		utils.Errorf("[Error] Cannot get response: %s", err)
 		return
 	}
-
 	defer func(Body io.ReadCloser) {
 		_ = Body.Close()
 	}(response.Body)
 
-	data, err := io.ReadAll(response.Body)
-	if err != nil {
-		utils.Errorf("[Error] Cannot read response: %s", err)
-		return
-	}
-
 	type Tag struct {
 		Ref string
 	}
 	var tags []Tag
+	var data []byte
+
+	goTagsCacheDir := os.TempDir()
```

Comment:
One more, we can use our gobrew directory for storing cache information. And this file will be stored  more secure. On more OS tmp directory clean automatically.

Reply:
yea 
I thought about using `.gobrew` dir, but then I thought that it is better to use the `/tmp` because of automatic cleanup there. And no need to pollute the `.gobrew` dir

Reply:
Although this caching is only there for `gobrew self-update` 
Because in order to get golang tags, we are now calling `raw.github` which is fine right now. But this implementation does cover for it.

Reply:
OK, got it